pyvista/core/utilities/reader.py

```diff
@@ -2774,11 +2776,167 @@ class FLUENTCFFReader(BaseReader):
     >>> mesh = blocks[0]
     >>> mesh.plot(cpos='xy', scalars='SV_T')
 
+    These Fluent result files typically contain several cell result arrays,
+    including:
+
+    >>> mesh.cell_data
+    pyvista DataSetAttributes
+    Association     : CELL
+    Active Scalars  : SV_T
+    Active Vectors  : None
+    Active Texture  : None
+    Active Normals  : None
+    Contains arrays :
+        SV_D                    float64    (4338,)
+        SV_DENSITY              float64    (4338,)
+        SV_DPM_PARTITION        float64    (4338,)
+        SV_H                    float64    (4338,)
+        SV_K                    float64    (4338,)
+        SV_MU_LAM               float64    (4338,)
+        SV_MU_T                 float64    (4338,)
+        SV_P                    float64    (4338,)
+        SV_T                    float64    (4338,)              SCALARS
+        SV_U                    float64    (4338,)
+        SV_V                    float64    (4338,)
+        SV_BF_V                 float64    (4338, 2)
+
+    For large files, you might want to turn off reading certain arrays to speed
+    up performance. If you're only interested in a single array, disable all
+    cell arrays and then enable just that one.
+
+    >>> reader = pv.get_reader(filename)
+    >>> reader.disable_all_cell_arrays()
+    >>> reader.enable_cell_array('SV_D')
+    >>> blocks = reader.read()
+    >>> mesh = blocks[0]
+    >>> mesh.cell_data
+    pyvista DataSetAttributes
+    Association     : CELL
+    Active Scalars  : None
+    Active Vectors  : None
+    Active Texture  : None
+    Active Normals  : None
+    Contains arrays :
+        SV_D                    float64    (4338,)
+
     """
 
     _vtk_module_name = 'vtkIOFLUENTCFF'
     _vtk_class_name = 'vtkFLUENTCFFReader'
 
+    @property
+    def cell_arrays(self) -> list[str]:
```

Comment:
The functionality here is more or less exactly the same as `PointCellDataSelection`, but just for the cell part.  I wonder if it makes sense to split it into `PointDataSelection` and `CellDataSelection` and then combine them into a wrapper `PointCellDataSelection`.  I thought there was another case of this in another PR, but can't find it quickly.

Reply:
If we don't do this, we should at least align on method naming, this one is https://docs.pyvista.org/api/readers/_autosummary/pyvista.pointcelldataselection.cell_array_names#pyvista.PointCellDataSelection.cell_array_names

Reply:
Also can this one be implemented for this reader https://docs.pyvista.org/api/readers/_autosummary/pyvista.pointcelldataselection.cell_array_status#pyvista.PointCellDataSelection.cell_array_status

Reply:
Realizing there are a bunch of attributes that likely could be added to from `PointCellDataSelection`.